.github/workflows/publish.yaml

```diff
@@ -73,6 +91,8 @@ jobs:
     timeout-minutes: 10
     env:
       GH_TOKEN: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
+      VERSION: ${{ github.event.inputs.version || github.ref_name }}
```

Comment:
Do we need this line since we set version in another step?

Reply:
I think so? Don't we need to set it in every step?

Reply:
Hmm double-checking..i see what you mean now

Reply:
Ah, you were right - nice catch. Removed!